sdk/web-pubsub/web-pubsub/rollup.base.config.js

```diff
@@ -0,0 +1,122 @@
+import path from "path";
```

Comment:
You might be able to get away with using the shared rollup.config like in the template package, but not a blocker:
https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/template/template/rollup.config.js

Reply:
Jeff mentioned same.